### DISCUSSION TITLE
bigfix/10.0/mcm/MCM/Config/c_admin_mg_mdrn_dev

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#20] [Q&A]
# bigfix/10.0/mcm/MCM/Config/c_admin_mg_mdrn_dev

This section guides you through BigFix Modern Client Management (MCM) and BigFix Mobile to understand the MCM concepts, terminologies, features, and functionality.

https://help.hcltechsw.com/bigfix/10.0/mcm/MCM/Config/c_admin_mg_mdrn_dev.html

**Comment by gayathripg** [comment]
Test
